src/Config/InfectionConfig.php

```diff
@@ -251,6 +258,12 @@ function ($excludeDir) use ($srcDir) {
 
     public function getTmpDir(): string
     {
-        return $this->config->tmpDir ?? sys_get_temp_dir();
+        $tmpDir = $this->config->tmpDir ?? sys_get_temp_dir();
+
+        if ($this->filesystem->isAbsolutePath($tmpDir)) {
```

Comment:
I'd say that we don't need to execute this line for `sys_get_temp_dir`.

so it's actually only when `$this->config->tmpDir` is not empty

Reply:
fixed